### PR TITLE
Login and signup forms.

### PR DESCRIPTION
**Pull request by JeldorPKU**
Modified the UI of login and signup, with style and controller changed. Now there are only views but no interaction between the front-end and the back-end. The buttons are set but most of them do not have any functions. Further navigation in the website is to be done in future PRs.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -27,7 +27,23 @@ body {
...
+    margin-bottom: 40px;
+    padding-top: 30px;
+    padding-left: 40px;
+    padding-right: 40px;
+    padding-bottom: 60px;
+    border-style: solid;
+    border-width: 1px;
+    border-color: #CBCBCB;
+}
+.regcontainer {
```
Comment (huxuan) [inline assets/css/style.css]: Add an empty line before new section.

File: assets/css/style.css [hunk 2]
```diff
@@ -27,7 +27,23 @@ body {
...
   //background-color: #c21a25;
   //background:url(/assets/images/test.jpg);
```
Comment (huxuan) [inline assets/css/style.css]: Let's start dropping this kind of useless snippets. ;-)

File: assets/css/style.css [hunk 3]
```diff
@@ -27,7 +27,23 @@ body {
   //background-color: #f3f3f3;
 }
 
-.mycontainer {
+.signcontainer {
+//background-color: #c21a25;
```
Comment (huxuan) [inline assets/css/style.css]: Remove the useless commented code.

File: application/views/signup_form.php [hunk 4]
```diff
@@ -1,23 +1,57 @@
...
+        <button class="btn btn-success btn-block">注册</button>
+    </div>
 
-<?php echo validation_errors(); ?>
 
-<?php
 
+</div>
+
+<!--
```
Comment (huxuan) [inline application/views/signup_form.php]: Remove the useless comment please.

**Comment by huxuan** [comment]
Almost great done. Just some small notes mostly for jiepi. ;-)

**Comment by JeldorPKU** [comment]
Shall I remove all the useless styles? They seem to be useless but I don't know whether we would be using them in the future.

**Pull request merged**